config/initializers/session_store.rb

```diff
@@ -0,0 +1,2 @@
+# Session store disabled as per devise-jwt setup at https://github.com/waiting-for-dev/devise-jwt
+Rails.application.config.session_store :disabled
```

Comment:
this is part of the devise-jwt instructions, and it says if you're using API only rails, then you most likely don't need sessions, so disable them.
https://github.com/waiting-for-dev/devise-jwt